src/openrct2/management/Award.cpp

```diff
@@ -77,11 +77,11 @@ bool award_is_positive(int32_t type)
 /** More than 1/16 of the total guests must be thinking untidy thoughts. */
 static bool award_is_deserved_most_untidy(int32_t activeAwardTypes)
 {
-    if (activeAwardTypes & (1 << PARK_AWARD_MOST_BEAUTIFUL))
+    if (activeAwardTypes & (1 << static_cast<int>(ParkAward::MostBeautiful)))
```

Comment:
I created a little utility function for this EnumsToFlags (EnumToFlag) it means you don't need the static cast and 1 <<.

Reply:
Oh okay thats sweet, I'll change that now

Reply:
Where is EnumToFlag? I can't seem to find it anywhere?

Reply:
I think it's on the yet to be merged #12833

Reply:
Yes, this PR is waiting for one more ok

Reply:
I have now switched `static_cast<int>` to `EnumToFlag`